spec/controllers/teams_controller_spec.rb

```diff
@@ -64,6 +64,56 @@
         end
       end
 
+      describe "#manage_users" do
+        let(:user1) { create :user, :with_team_and_is_admin }
+        let(:user2) { create :user, :with_team_and_is_admin }
+        it "list users this team to manage" do
+          get :manage_users, team_id: team.slug
+          expect(response).to be_success
+          expect(assigns[:users]).to eq([user])
+        end
+      end
+
+      describe "#associate_user" do
+        before { sign_in user }
+        context "with valid parameters" do
+          let(:values) do
+            {
+              email: 'sample@example.com',
+            }
+          end
+
+          context "when user find no have team" do
+            let!(:normal_user_without_team) { create :user, email: "sample@example.com" }
+            it "should return successfully associate in current team" do
+              post :associate_user, team_id: team.slug, user: values
+
+              expect(response).to redirect_to(team_find_user_by_email_path)
+              expect(flash[:notice]).to eq(I18n.t('teams.team_was_successfully_updated'))
+            end
+          end
+
+          context "when user already in this team" do
```

Comment:
I think `when user is in the current team` is better, the user may be on a team already but not be the current one.